### ISSUE TITLE
Proposal to obsolete GO:0042624 ATPase activity, uncoupled

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#16971]
The proposal has been made to obsolete "GO:0042624 ATPase activity, uncoupled"

The reason for obsoletion is that the there is no convincing example of a protein with this function. 
- There is a single EXP annotation by SGD (S000000838), and looking at the paper (PMID:16260602), there is actually evidence of a coupled function.  
- The poster child example given in the term definition (SERCA- BTW, not annotated to the term) cites an early paper (PMID:12912988 "Brown adipose tissue Ca2+-ATPase: uncoupled ATP hydrolysis and thermogenic activity.)". A later paper, PMC3446692, confirms that the protein has a coupled activity,   in fact it is a rather inefficient ATP-coupled Ca+ pump. 

Any comments can be  added to the issue: https://github.com/geneontology/go-ontology/issues/16971

We are opening a comment period for this proposed obsoletion. We’d like to proceed and obsolete this term on March 11th, 2019. Unless objections are received by March 11th, 2019.  we will assume that you agree to this change.

Thanks, Pascale

**Comment by pgaudet** [comment]
Once the obsoletion is done, ATP-dependent helicase activity will be merged into 'helicase activity' - same for RNA/DNA siblings.

**Comment by pgaudet** [comment]
sent email

**Comment by srengel** [comment]
i edited the one SGD annotation.  so now there are no EXP annotations.

**Comment by pgaudet** [comment]
Thanks @srengel
